decider-sample-fingerprinting/src/main/java/ca/on/oicr/pde/deciders/SampleFingerprintingDecider.java

```diff
@@ -36,8 +36,6 @@
 public class SampleFingerprintingDecider extends OicrDecider {
 
     private final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.S");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "DATE_FORMAT" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)